.buildkite/engineer

```diff
@@ -13,7 +13,7 @@ fi
 if ! type "engineer" > /dev/null; then
     # Setup Prisma engine build & test tool (engineer).
     set -e
-    curl --fail -sSL "https://prisma-engineer.s3-eu-west-1.amazonaws.com/lrts/latest/$OS/engineer.gz" --output engineer.gz
+    curl --fail -sSL "https://prisma-engineer.s3-eu-west-1.amazonaws.com/buh-bye-clippy/latest/$OS/engineer.gz" --output engineer.gz
```

Comment:
Both old and new urls look like they're not what they should be on master — it's going to change when the engineer PR is merged, right?

Reply:
Yeah the original branch was a surprise. I started mine from that, and will do a new engineer release before merging.

Reply:
Fixed in: https://github.com/prisma/prisma-engines/pull/2158/commits/3da8692d9dac8ebbe2d3a49163237af069847cbe